odxtools/parameters/reservedparameter.py

```diff
@@ -1,27 +1,35 @@
 # SPDX-License-Identifier: MIT
 import warnings
+from dataclasses import dataclass
 
 from ..decodestate import DecodeState
 from ..exceptions import DecodeError
 from .parameter import Parameter
 
 
+@dataclass
 class ReservedParameter(Parameter):
-
-    def __init__(self, *, bit_length, **kwargs):
-        super().__init__(parameter_type="RESERVED", **kwargs)
-        self._bit_length = bit_length
+    bit_length_raw: int
 
     @property
-    def bit_length(self):
-        return self._bit_length
+    def parameter_type(self) -> str:
+        return "RESERVED"
 
     def is_required(self):
         return False
 
     def is_optional(self):
         return False
 
+    @property
+    def bit_length(self) -> int:
+        # this is a bit hacky: the parent class already specifies
+        # bit_length as a function property, and we need to change
+        # this to return the value from the XML here. Since function
+        # attributes cannot be overridden by non-function ones, we
+        # need to take the "bit_length_raw" detour...
```

Comment:
Another approach is to explicitly make an `__init__` in this class

Reply:
yes, the problem with this is that I don't know how the constructor of the dataclass can be called. (I don't want to have to explicitly handle all attributes...)